src/DGmethods/DGBalanceLawDiscretizations.jl

```diff
@@ -659,6 +688,64 @@ function grad_auxiliary_state!(disc::DGBalanceLaw, id, (idx, idy, idz))
                            Dmat, topology.elems, id, idx, idy, idz))
 end
 
+
+"""
+    indefinite_stack_integral!(disc, f, Q, out_states, [P=disc.auxstate])
+
+Computes the indefinite integral along an element stack using state `Q`
+```math
+∫_{ζ_{0}}^{ζ} f(q; aux, t)
```

Comment:
Is this the integral from the base upwards? I think in some cases they need the integral in the other direction

Reply:
Yes it is. Given the top value you can extract the other one... that said we do not have convenient way to access this at the user level.

Reply:
Could change this to:
```julia
indefinite_stack_integral!(disc::DGBalanceLaw, f, Q, out_states,
                                    P=disc.auxstate; direction=:ascending)
```
or something like that.

Reply:
How would one `accumulate` backward in julia? This would really be the analog in summation case.

Reply:
One option would be to have a reverse a precomputed indefinite integral function if up is the standard way, but sometimes backward is also needed.

Reply:
I've added a `reverse_indefinite_stack_integral!` function